### PR TITLE
Include Teleport agent installation instructions for App,DB Guides

### PR DESCRIPTION
**Pull request by stevenGravy**
The DB, app guides and getting started did not include the step to install Teleport on the host running the db service.  The example is on a server so makes sense to include the linux install instrs.

### CODE REVIEW

**Comment by benarent** [comment]
I feel adding a diagram would really help to explain this concept.  I added these two to my recent Redis Blog post https://goteleport.com/blog/access-redis/ 
image

**Review by r0mant — APPROVED** [review]

File: docs/pages/application-access/getting-started.mdx [hunk 1]
```diff
@@ -62,8 +62,10 @@ $ tctl tokens add \
 
 ### Start Teleport
 
-On the host where you will run the Teleport Application Service, download the latest version of Teleport for your platform from our
-[downloads page](https://goteleport.com/teleport/download).
+
+On the host where you will run the Teleport Application Service install Teleport. See our [Installation](/docs/installation/) page for other options besides Linux servers.
```
Comment (ptgott) [inline docs/pages/application-access/getting-started.mdx:66]:
```suggestion
Install Teleport on the host where you will run the Teleport Application Service. See our [Installation](/docs/installation/) page for options besides Linux servers.
```
Style tweak

File: docs/pages/application-access/guides/connecting-apps.mdx [hunk 2]
```diff
@@ -97,6 +97,10 @@ $ tctl --config=/path/to/teleport.yaml users add --roles=access appuser
 
 ## Start application service with CLI flags
 
+Install Teleport
```
Comment (ptgott) [inline docs/pages/application-access/guides/connecting-apps.mdx:100]:
```suggestion
Install Teleport:
```

File: docs/pages/database-access/getting-started.mdx [hunk 3]
```diff
@@ -99,6 +99,9 @@ see the AWS [documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGui
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/getting-started.mdx:102]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx [hunk 4]
```diff
@@ -47,6 +47,10 @@ This guide will help you to:
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:50]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/cockroachdb-self-hosted.mdx [hunk 5]
```diff
@@ -38,6 +38,12 @@ This guide will help you to:
 ### Set up the Teleport Database Service
 
 (!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/cockroachdb-self-hosted.mdx:42]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/mongodb-atlas.mdx [hunk 6]
```diff
@@ -30,6 +30,10 @@ In this guide you will:
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/mongodb-atlas.mdx:33]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/mongodb-self-hosted.mdx [hunk 7]
```diff
@@ -38,6 +38,10 @@ MongoDB instances.
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/mongodb-self-hosted.mdx:41]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/mysql-cloudsql.mdx [hunk 8]
```diff
@@ -108,6 +108,10 @@ Teleport Database Access for Cloud SQL MySQL is available starting from the
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/mysql-cloudsql.mdx:111]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/mysql-self-hosted.mdx [hunk 9]
```diff
@@ -25,6 +25,10 @@ Teleport Database Access for MySQL is available starting from Teleport version
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/mysql-self-hosted.mdx:28]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/postgres-cloudsql.mdx [hunk 10]
```diff
@@ -178,6 +178,10 @@ the `6.2` Teleport release.
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/postgres-cloudsql.mdx:181]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/postgres-redshift.mdx [hunk 11]
```diff
@@ -27,6 +27,10 @@ videoBanner: UFhT52d5bYg
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/postgres-redshift.mdx:30]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/postgres-self-hosted.mdx [hunk 12]
```diff
@@ -25,6 +25,10 @@ release.
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/postgres-self-hosted.mdx:28]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/rds.mdx [hunk 13]
```diff
@@ -39,6 +39,10 @@ This guide will help you to:
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/rds.mdx:42]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/redis-cluster.mdx [hunk 14]
```diff
@@ -48,6 +48,10 @@ This guide will help you to:
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/redis-cluster.mdx:51]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/redis.mdx [hunk 15]
```diff
@@ -50,6 +50,10 @@ This guide will help you to:
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/redis.mdx:53]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/database-access/guides/sql-server-ad.mdx [hunk 16]
```diff
@@ -212,6 +212,10 @@ KVNO Principal
 
 (!docs/pages/includes/database-access/token.mdx!)
 
+Install Teleport into host for Teleport Database Service
```
Comment (ptgott) [inline docs/pages/database-access/guides/sql-server-ad.mdx:215]:
```suggestion
Install Teleport on the host where you will run the Teleport Database Service:
```

File: docs/pages/application-access/getting-started.mdx [hunk 17]
```diff
@@ -62,8 +62,10 @@ $ tctl tokens add \
 
 ### Start Teleport
 
-On the host where you will run the Teleport Application Service, download the latest version of Teleport for your platform from our
-[downloads page](https://goteleport.com/teleport/download).
+
+Install Teleport on the host where you will run the Teleport Application Service. See our [Installation](/docs/installation/) page for options besides Linux servers.
```
Comment (ptgott) [inline docs/pages/application-access/getting-started.mdx:66]: The `Installation` link should lead to an .mdx page so our NextJS tooling can fill in the `/ver/<version>` path segments in the URL as needed.

Comment (ptgott) [reply docs/pages/application-access/getting-started.mdx:66]: Also looks like there are some outstanding suggestions.

Comment (stevenGravy) [reply docs/pages/application-access/getting-started.mdx]: addressed

**Comment by stevenGravy** [comment]
have added diagrams for each db and the app getting started. @ptgott believe all items have been addressed.

**Review by ptgott — APPROVED** [review]
I think the diagrams work. There are still a few instances where the docs say, "Install Teleport into host for Teleport Database Service." I've disabled auto-merge so you can correct the grammar, but approved this so you're not blocked.

**Comment by stevenGravy** [comment]
> 

thanks @ptgott , didn't meant to miss those install changes.  Updated.

**Pull request merged**